sound/attributions.txt

```diff
@@ -1,4 +1,7 @@
 effects/creatures/vannatusk_attack.ogg - "Monster 2.wav" by Sea Fury, licensed under CC BY 3.0. Obtained from https://freesound.org/people/Sea%20Fury/sounds/48673/
 effects/creatures/vannatusk_sound.ogg  - "Monster 3.wav" by Sea Fury, licensed under CC BY 3.0. Obtained from https://freesound.org/people/Sea%20Fury/sounds/48674/
 effects/creatures/vannatusk_sound_2.ogg - "Monster 6.wav" by Sea Fury, licensed under CC BY 3.0. Obtained from https://freesound.org/people/Sea%20Fury/sounds/48971/
-No changes were made to the sounds, and all credit goes to Sea Fury
\ No newline at end of file
+No changes were made to the sounds, and all credit goes to Sea Fury
+
+https://github.com/Aurorastation/Aurora.3/pull/14169 - All sounds used part of "Valhalla", licensed under CC0 1.0 Universal (CC0 1.0). Obtained from https://freesound.org/people/FilmmakersManual/
```

Comment:
You need to specify the files added by your PR instead of linking the PR.

Reply:
Oh. Alright. I will check out the readme and fix this.